src/intervals/rounding.jl

```diff
@@ -9,6 +9,7 @@ interval rounding types, e.g.
 +(IntervalRounding{:none}, a, b, RoundDown)
 
 The current allowed rounding types are
+- :errorfree # use errorfree arithmetic via FastRounding.jl
```

Comment:
I suggest the following changes for naming the rounding modes: 
- `:errorfree` should be named `:fast`
-`:fast` should be named `:wider` (or `:wide`?)